compiler/tests/schema/constructor_parameters_fields.bond

```diff
@@ -0,0 +1,44 @@
+namespace Test
```

Comment:
Since the two .bond files are identical, take a look at what was done for collection-interfaces to put the expected files in a different directory.